aspnet/dnx/commands.rst

```diff
@@ -1,8 +1,232 @@
+
+Global commands use the local XML *NuGet.Config* file to store package source locations. The main sections for this file are ``packageRestore``, ``packageSources``, ``disabledPackageSources``, and ``activePackageSource``.
+
+Installing global commands
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can add a global command package and its dependencies using the `Package Manager Console <http://docs.nuget.org/consume/package-manager-console>`_. For example, to install the `SecretManager <http://www.nuget.org/packages/Microsoft.Framework.SecretManager>`_ from the **Package Manager Console**, enter the following::
+
+	Install-Package Microsoft.Framework.SecretManager -Pre
+	
+.. note:: The global *NuGet.config* file is used to find the correct NuGet feed when installing global command NuGet packages. Use ``Install-Package -?`` from the **Package Manager Console** to view help information related to the ``Install-Package`` command. 
+
+You can then use the .NET Development Utility (DNU) to install the commands contained in the package. For example, enter the following from the command prompt::
+
+	dnu commands install Microsoft.Framework.SecretManager
+	
+.. note:: You can use the ``--overwrite`` option to overwrite conflicting commands. Use ``dnu commands install -?`` from the command prompt to view help information related to the ``install`` command.
+
+Running global commands
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You can run global commands from the command prompt after installing the related package. For example, if you have installed the SecretManager and have set the user secret for the application, from the application directory you can issue the following command to retrive all of the user secrets for your application::
+
+	user-secret list
+	
+.. note:: To see a list of the available DNX runtimes, including the **active** DNX runtime, you can enter ``dnvm list`` from the command prompt. If you need to change the active DNX runtime, use ``dnvm use [version] -p``. For example, ``dnvm use 1.0.0-beta6 –p``. Global commands always run with the active DNX runtime. 
+	
+Uninstalling global commands
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To uninstall global commands you can use the following DNX command::
+
+	dnu commands uninstall [arguments] [options]
+	
+The [arguments] is the name of the command to uninstall. For example::
+
+	dnx commands uninstall Microsoft.Framework.SecretManager
+	
+For additional details about the uninstall command, enter ``dnu commands uninstall -?`` from the command prompt.	
+
+Built-in global commands
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following built-in global commands are available: 
+
+	1. user-secrets
+	2. sqlservercache
+
+These commands have specific NuGet packages that must be installed. Once a global command package is installed, you can install the command using the DNU. 
```

Comment:
You don't need to install the package to install the global command. You can install it directly